tests/unit_test/callback/pyscript_handler_test.py

```
@@ -3852,6 +3905,57 @@ def test_save_as_pdf_error_raises_wrapped_exception():
             filepath="report.pdf"
         )
 
+def test_typing_indicator_success():
```

Ruff (0.14.5)

Redefinition of unused `test_typing_indicator_success` from line 2050

(F811)